content/docs/introduction/faq.md

```diff
@@ -211,37 +225,35 @@ after over 285 years.
 ### Why does Prometheus use a custom storage backend rather than [some other storage method]? Isn't the "one file per time series" approach killing performance?
 
 Initially, Prometheus ran completely on LevelDB, but to achieve better
-performance, we had to change the storage for bulk sample data. We
-evaluated many storage backends that were available at the time,
-without getting satisfactory results. So we implemented exactly the
-parts we needed, while keeping LevelDB for indexes and making heavy
-use of file system capabilities. Obviously, we could not evaluate
-every single storage backend out there, and storage backends have
-evolved meanwhile. However, the performance of the solution
-implemented now is satisfactory for most use-cases. Our most important
-requirements are an acceptable query speed for common queries and a
-sustainable ingestion rate of many thousands of samples per
-second. The latter depends on the compressibility of the sample data
-and on the number of time series the samples belong to, but to give
-you an idea, here are some results from benchmarks:
-
-* On an older 8-core machine with Intel Core i7 CPUs, 8GiB RAM, and
-  two spinning disks (Samsung HD753LJ) in a RAID-1 setup, Prometheus
-  sustained an ingestion rate of 34k samples per second, belonging to
-  170k time series, scraped from 600 targets.
-
-* On a modern server with 64GiB RAM, 32 CPU cores, and SSD, Prometheus
-  sustained an ingestion rate of 525k samples per second, belonging to 1.4M
-  time series, scraped from 1650 targets.
-
-In both cases, there were no obvious bottlenecks. Various stages of the
-processing pipelines reached their limits more or less at the same
-ingestion rate.
-
-Running out of inodes is highly unlikely in a usual set-up. There is a
-possible downside: If you want to delete Prometheus's storage
-directory, you will notice that some file systems are very slow when
-deleting files.
+performance, we had to change the storage for bulk sample data. We evaluated
+many storage backends that were available at the time, without getting
+satisfactory results. So we implemented exactly the parts we needed, while
+keeping LevelDB for indexes and making heavy use of file system
+capabilities. Obviously, we could not evaluate every single storage backend out
+there, and storage backends have evolved meanwhile. However, the performance of
+the solution implemented now is satisfactory for most use-cases. Our most
+important requirements are an acceptable query speed for common queries and a
+sustainable ingestion rate of many thousands of samples per second. The latter
```

Comment:
Yes, why not. (Incidentally, I checked out the original storage v2 design docs recently, and there we asked for just thousands of samples per second... the original storage v1 was really slow... ;)